packages/web/components/Modal/SettleDealModal/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+import { ReviewButton as Button } from '../CurateTechnologyModal/styles';
+import { toast } from '../../Toast';
+import { Modal, InfosContainer, Summary } from './styles';
+import { CurrencyInputField, InputField } from '../../Form';
+import { formatCurrencyToInt, formatMoney } from '../../../utils/helper';
+
+const SettleDealModal = ({ closeModal }) => {
+	const [totalValue, setTotalValue] = useState(0);
+	const form = useForm();
+	const { watch } = form;
+	const formValues = watch();
+
+	const onSubmit = () => {
+		toast.success('Pedido fechado com sucesso');
+		closeModal();
+	};
+
+	useEffect(() => {
+		const { quantity, unityPrice } = formValues;
+
+		if (quantity && unityPrice) {
+			const intCurrency = formatCurrencyToInt(unityPrice);
+			const total = intCurrency * quantity;
+
+			setTotalValue(total);
+		}
+	}, [formValues]);
+
+	return (
+		<Modal onSubmit={form.handleSubmit(onSubmit)} noValidate>
+			<div>
+				<img
+					src="/checkout-rafiki.svg"
+					alt="Ilustração de uma moça registrando compras em um computador"
+				/>
+			</div>
+
+			<InfosContainer>
+				<h3>Deseja fechar este pedido?</h3>
+
+				<CurrencyInputField
+					form={form}
+					label="Valor unitário negociado"
+					name="unityPrice"
+					placeholder="R$ 0,00"
+					variant="gray"
+					validation={{ required: true }}
+				/>
+
+				<InputField
+					form={form}
+					name="quantity"
+					placeholder="Quantidade negociada"
+					label="Quantidade negociada"
+					validation={{
+						required: true,
+						pattern: {
+							value: /^[0-9]*$/,
+							message: 'Você deve digitar apenas números positivos',
+						},
+					}}
+					type="number"
+					min="0"
+					variant="gray"
+				/>
```

Comment:
Here's another suggestion to improve UI on mobile:
![image](https://user-images.githubusercontent.com/10481028/98876773-bb6d6800-245d-11eb-9a4a-6ff91117a7a6.png)
It looks a bit weird with these labels and inputs smaller than the action buttons.
Maybe this could look a bit better:
![image](https://user-images.githubusercontent.com/10481028/98876829-dd66ea80-245d-11eb-86f0-8240a80ed571.png)